app/src/main/java/com/example/desafioconcrete/Adapter/AdapterDetails.kt

```diff
@@ -18,17 +18,17 @@ class AdapterDetails(
 
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
         var detail = detailsList[position]
-        var title = holder.itemView.lblTitleName
+        var title = holder.itemView.hint3
         var description = holder.itemView.lblDescricaoRepositorioDetails
-        var username = holder.itemView.lblUsernameDetails
+        var username = holder.itemView.hint2
```

Comment:
Why are the variables called with numbers?

Reply:
It's only for a loading activity